purchase_cancel_reason/tests/test_purchase_cancel_reason.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013 Guewen Baconnier, Camptocamp SA
+# Copyright 2017 Okia SPRL
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from odoo import fields
+from odoo.tests.common import TransactionCase
+
+
+class TestPurchaseCancelReason(TransactionCase):
```

Comment:
We could inherit from `BaseCommon` class to disable tracking on all records created in tests.